templates/openstack/consulting.html

```diff
@@ -12,601 +18,564 @@
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Repeatable artifacts</h3>
-            <p class="p-matrix__desc">
-              At the end of the engagement, we provide you with a detailed deployment guide that enables repeatable re-deployments of your private cloud and perform a formal handover process.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Repeatable artifacts</h3>
+          <p>
+            At the end of the engagement, we provide you with a detailed deployment guide that enables repeatable re-deployments of your private cloud and perform a formal handover process.
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Day-2 automation</h3>
-            <p class="p-matrix__desc">
-              With PCB, you are not left alone on day-2. Charmed OpenStack provides full automation around common day-2 maintenance tasks, including daily operations, scale-out capabilities, integration and OpenStack upgrades.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Day-2 automation</h3>
+          <p>
+            With PCB, you are not left alone on day-2. Charmed OpenStack provides full automation around common day-2 maintenance tasks, including daily operations, scale-out capabilities, integration and OpenStack upgrades.
+          </p>
         </li>
-        <li class="p-matrix__item">
-          <div class="p-matrix__content">
-            <h3 class="p-matrix__title">Optional training</h3>
-            <p class="p-matrix__desc">
-              On request, we train your personnel to get them up to speed with your new private cloud. We provide comprehensive training courses that cover OpenStack architecture, implementation, and operations.
-            </p>
-          </div>
+        <li class="p-list__item">
+          <h3 class="p-heading--5">Optional training</h3>
+          <p>
+            On request, we train your personnel to get them up to speed with your new private cloud. We provide comprehensive training courses that cover OpenStack architecture, implementation, and operations.
+          </p>
         </li>
       </ul>
-      <p class="u-sv3">
-        <a class="p-button--positive"
-           href="https://assets.ubuntu.com/v1/195a305f-Private%20Cloud%20Build_DS_27.06.2024%20update.pdf">Download PCB datasheet</a>
-      </p>
+    </div>
+    <div class="row p-section">
+      <div class="col-start-large-7 col-6">
+        <hr class="p-rule--muted" />
+        <a href="https://assets.ubuntu.com/v1/195a305f-Private%20Cloud%20Build_DS_27.06.2024%20update.pdf">Download PCB datasheet&nbsp;&rsaquo;</a>
```

Comment:
This link doesn't appear in the copydoc or the design

Reply:
Elaine stated that
> Asked Tytus about the data sheet to link - we were under instructions to follow what's on the live page in case of discrepancies, but I tagged him in a comment anyway to double check

[here](https://github.com/canonical/ubuntu.com/pull/15356#issuecomment-3084484407)

And Tytus also commented on the link in the copydoc

The agreement was that the one on prod would be used. I think the copydoc needs to be updated